### DISCUSSION TITLE
Percy not running on the latest browser versions

### DISCUSSION THREAD

**Discussion by skambo** [percy/cli#829] [Support] [answered]
We're running `percy/cli": "1.0.0-beta.76"` however, our Percy builds are running on these browser versions.

<img width="177" alt="percy browsers" src="https://user-images.githubusercontent.com/4996911/158367456-21fa7d98-cc6e-49bb-affb-90bc3cf2fba8.png">

rather than the latest browser versions available i.e. 

```
    Safari 15
    Chrome 98
    Edge 99
    Firefox 98
```

Percy docs read as `Percy currently supports modern Chrome, Firefox, Edge and Safari browsers, with support for more browsers in the works. We automatically handle browser upgrades, versioning, and maintenance in our custom rendering infrastructure` 

We're aiming to target latest browser versions - is there something amiss or that we need to do? Thank you

**Comment by Robdel12** [comment, marked as answer]
Hey @skambo! This isn't really an SDK issue or concern (we don't control the infrastructure). Those are currently the browsers we are using in our rendering infrastructure. I know there's currently work to upgrade the browsers but I don't have an ETA on when or what versions we'll be upgrading to. 

Might be worth reaching out to support@percy.io to get a more specific answer
